Sources/screen-capture-permissions/ScreenCapturePermissions.swift

```diff
@@ -2,18 +2,16 @@ import Cocoa
 
 struct ScreenCapturePermissions {
   static func checkPermissions() -> String {
-    guard let stream = CGDisplayStream(
+    let stream = CGDisplayStream(
       dispatchQueueDisplay: CGMainDisplayID(),
       outputWidth: 1,
       outputHeight: 1,
       pixelFormat: Int32(kCVPixelFormatType_32BGRA),
       properties: nil,
       queue: DispatchQueue.global(),
-      handler: { status, displayTime, frameSurface, updateRef in })
-    else {
-      return "false";
-    }
+      handler: nil
+    )
 
-    return "true";
+    return stream == nil ? "false" : "true"
```

Comment:
Generally, you should create APIs that are not made specifically for CLIs. That makes the code nicer. So this method should return a `Bool`. Then the above could be:
```suggestion
    return stream != nil
```